abstract.md

```diff
@@ -0,0 +1,41 @@
+# Making research software more robust
+Morgan Taschuk and Greg Wilson
+
+
+At the cutting edge of research, bioinformatics software is often written by a
+single person who is both developer and scientist, producing insightful, novel
+results for their own projects. Upon presenting their work, other
+people will want to use their methods. But much of academic software is written
+in a prototypic, exploratory and rigid style that is incompatible with reuse. Often, the end user will
```

Comment:
"in an ad hoc, exploratory style" (I don't think most people will understand "prototypic", and the sense of "rigid" won't be clear until you've introduced "robust" as contrast)